src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/repository/impl/ords/DisputeRepositoryImpl.java

```diff
@@ -101,7 +103,22 @@ public Iterable<Dispute> findAll() {
 
 	@Override
 	public Optional<Dispute> findById(Long id) {
-		throw new NotYetImplementedException();
+		if (id == null) {
+			throw new IllegalArgumentException("Dispute ID is null.");
+		}
+		try {
+			ViolationTicket violationTicket = violationTicketApi.v1ViolationTicketGet(null, Long.toString(id.longValue()));

```

Comment:
Just thinking that maybe you could update the yaml so that the disputeID is an int64 like I did for the delete endpoint. That way you don't have to do any type conversions.

Reply:
Yeah, that's a good idea actually. Will change that in the yaml. Thanks!